src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameTests.cs

```diff
@@ -458,6 +458,7 @@ public void TestSignaturesV2_3_X()
             _df.Explain(false);
 
             Assert.Equal(2, _df.Columns().ToArray().Length);
+            Assert.IsType<string[][]>(_df.DTypes());

```

Comment:
@imback82 , @suhsteve  Is this signature test enough, or do we want to add a separate test testing the actual functionality of `DTypes()`?